metadata-load/src/test/java/uk/ac/ebi/ampt2d/metadata/importer/MetadataImporterMainApplicationDBTest.java

```diff
@@ -52,21 +53,25 @@
     @Autowired
     private AnalysisRepository analysisRepository;
 
+    @Autowired
+    private SampleRepository sampleRepository;
+
     @Test
     @Category(OracleDbCategory.class)
     public void run() throws Exception {
         metadataImporterMainApplication.run(new DefaultApplicationArguments(
                 new String[]{"--accessions.file.path=analysis/EgaAnalysisAccessions.txt"}));
         assertEquals(2, studyRepository.count());
         assertEquals(6, analysisRepository.count());
-
+        assertEquals(6, sampleRepository.count());
         sraObjectsImporterThroughDatabase.getAccessionsToStudy().clear();
 
         // Import analysis having shared study that is already imported before
```

Comment:
Will previous count be continued for below asserts?

Reply:
Yes, the counts are supposed to be continued in this test. First we import 6 analyses and 6 samples which come from 2 studies. Then in the second run we import additional 5 analyses and 5 samples from the same 2 studies. Finally, we check that each of the studies has been imported only once, and counts of analyses and files also add up

Reply:
```suggestion
        // Import additional analyses into already imported study
```

Reply:
Fixed in https://github.com/EBIvariation/metadata-ws/pull/55/commits/886001aa5f38f2be3bb07ba5c1ceb550cf467ade